vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -251,6 +252,12 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 	if errAttach != nil {
 		log.WithFields(log.Fields{"name": r.Name,
 			"error": errAttach}).Error("Attach volume failed, removing the volume ")
+		if strings.Contains(errAttach.Error(), recvError) {
```

Comment:
>  is returning from a recv() call without actually waiting for data from the vSphere service.

are you saying we simply need to wait in vmci_client.c until more data arrives ?